src/main/kotlin/com/romainpiel/catsanddogs/Main.kt

```diff
@@ -23,10 +24,51 @@ fun main(args: Array<String>) {
             get("/schedule.json") {
                 val fromStr: String? = call.request.queryParameters["from"]
                 val from = fromStr?.let { OffsetDateTime.parse(it) }
+                val locale = Locale.forLanguageTag("pl")
```

Comment:
Nice, I think we could pass that local as a `Accept-Language` header

Reply:
I've done what I think needs to be done, but it won't work. :(

Reply:
I'm curious why it won't work, I've just tested and it works for me

Reply:
Oh retrofit 🤔 let me see...

Reply:
So I have just tested and it looks fine to me:
```
GET /schedule.json
Accept-Language: en-UK

[
    {
        "title": "We are counting on you",
        "subtitle": "Jeff Watkins",
        "date": "Monday, 8 May 2017",
        "time": "10:00"
    },
```

```
GET /schedule.json
Accept-Language: fr-FR

[
    {
        "title": "We are counting on you",
        "subtitle": "Jeff Watkins",
        "date": "Lundi, 8 mai 2017",
        "time": "10:00"
    },
```